lib/plugins/browsertime/index.js

```diff
@@ -271,10 +271,17 @@ module.exports = {
           .tap(results => {
             if (results.screenshots) {
               queue.postMessage(
-                make('browsertime.screenshot', results.screenshots, {
-                  url,
-                  group
-                })
+                make(
+                  'browsertime.screenshot',
+                  {
```

Comment:
This is technically a backwards incompatible change (external plugins would need to change as we do). I think it's ok, just need to include a mention in the changelog (in the PR would be good).

A good reminder as well to audit all other plugins to always use a top level object for message data, to allow for further expansion without similar api breakage.

Reply:
hmm that's true we should do it in another way.

Reply:
I've changed it now so that HTML listen on the browsertime config.